### ISSUE TITLE
import/no-unresolved triggers for import of browserslist-useragent-regexp

### ISSUE THREAD

**Issue by grahamb** [import-js/eslint-plugin-import#2656]
I am using `browserslist-useragent-regexp` in my project. When I attempt to import one of its exported functions in my project, eslint throws an error for `import/no-unresolved`.

## Versions:
eslint@8.31.0
eslint-plugin-import@2.26.0

## Usage:

```js
import { getUserAgentRegex } from 'browserslist-useragent-regexp';
```

## Expected result:

eslint should not mark the import as an error

## Actual result:

eslint throws a `imoprt/no-unresolved` error with message "Unable to resolve path to module 'browserslist-useragent-regexp'"

![image](https://user-images.githubusercontent.com/146111/211653293-c7b1bdea-18f5-4782-8811-5d7a3028bbd8.png)

**Comment by ljharb** [comment]
That's because v4 of browserslist-useragent-regexp doesn't provide a "main", only an "exports", and thus isn't backwards-compatible with older nodes, or with lots of the ecosystem. Thus plugin depends on `resolve`, which does not yet support `exports`.

I'd suggest downgrading to v3, or filing an issue on the package asking for a CJS "main" to be provided - otherwise, until `resolve` supports `exports`, you'll need to configure the rule to ignore this package, or, you'll need to find a resolver that supports `exports`.
